libraries/AP_Scripting/docs/docs.lua

```diff
@@ -2387,6 +2387,10 @@ function vehicle:is_taking_off() end
 ---@return boolean
 function vehicle:is_landing() end
 
+-- desc
+---@return integer
+function vehicle:get_landed_state() end
```

Comment:
Full descriptions please. 

You can enumerate the return states like this:

https://github.com/ArduPilot/ardupilot/blob/eb8eeba8516f5fc3a83b82a7f03235c2ca6578ae/libraries/AP_Scripting/docs/docs.lua#L1416-L1423